pymedphys/_dicom/anonymise/__init__.py

```diff
@@ -34,13 +34,22 @@
 )
 from pymedphys._dicom.utilities import remove_file
 
-from . import strategy
+from . import strategies
 
 HERE = dirname(abspath(__file__))
 
 IDENTIFYING_KEYWORDS_FILEPATH = pjoin(HERE, "identifying_keywords.json")
 
 
+def anonymise(ds, copy=True, strategy=None):
+    if strategy is None:
+        return _anonymise_dataset(ds, copy_dataset=copy)
+
+    return _anonymise_dataset(
+        ds, copy_dataset=copy, replacement_strategy=strategy, **strategy
```

Comment:
this exposes each VR string as a variable (i.e. DS becomes a variable).
I get that this is a compact representation, but it leans towards implicit instead of explicit.

Reply:
Yeah, this was more of a swiss army knife approach to allow the API to be refactored without actually changing anything and then replaced later. But I agree, this is implicit rather than explicit, and verbosity here doesn't hurt.